File: src/main/java/org/team1/nbe1_3_team01/domain/user/service/response/UserBriefWithRoleResponse.java
package org.team1.nbe1_3_team01.domain.user.service.response;

import org.team1.nbe1_3_team01.domain.user.entity.Role;

public record UserBriefWithRoleResponse(
        Long id,
        String username,
        String name,
        Role role
) {
}

File: src/main/java/org/team1/nbe1_3_team01/domain/user/service/response/UserBriefWithRoleResponse.kt
package org.team1.nbe1_3_team01.domain.user.service.response

import org.team1.nbe1_3_team01.domain.user.entity.Role


data class UserBriefWithRoleResponse(
    val id: Long,
    val username: String,
    val name: String,
    val role: Role
)
